test/jdk/java/awt/image/multiresolution/MultiresolutionIconTest.java

```diff
@@ -128,7 +128,7 @@ private void UI() {
 
     private boolean checkPressedColor(int x, int y, Color ok) {
 
-        r.mouseMove(x, y);
+        r.mouseMove(x+5, y+5);
```

Comment:
What's the purpose of moving the robot mouse move +5 in the x and y directions if the fix is to skip the test in MacOS?

Reply:
The test isn't being skipped on macOS, only AquaL&F. The mouse move +5 is a workaround for the test failing on macOS-aarch64 systems because of a cursor zoom on mouse move issue.

Reply:
Got it. Poor phrasing on my part; meant Aqua L&F as you mentioned. I was just curious about the additional change to the x & y.